internal/pkg/service/templates/api/gen/http/templates/server/encode_decode.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by goa v3.11.1, DO NOT EDIT.
+// Code generated by goa v3.14.6, DO NOT EDIT.
 //
 // templates HTTP server encoders and decoders
 //
```

Comment:
This commit contains generated code, updated by the `make generate-templates-api`.

Now there are few changes, but we are not used to reviewing this code.

We just went over it quickly to see if there is a obvious problem.

Templates API E2E tests are passing without modification and cover all operations.